src/db/index.js

```diff
@@ -19,16 +19,31 @@ const db = {
  * Be sure to wrap each migration script in a try/catch and throw on Error.
  * We don't want the server to run if the migration failed.
  */
-async function applyMigrations() {
+async function applyMigrations(db) {
   // Run migration scripts here
+  const migrationPath = require('path').join(__dirname, 'migrations');
+  const migrations = [];
+  try {
+    await fs.readdirSync(migrationPath).forEach(async (file) => {
```

Comment:
shouldn't have to await `readdirSync`. it is synchronous.

Reply:
we need to ensure after reading the files in /migrations that they are in sorted order as well. we want to make sure 1, 2, 3, etc ordering to ensure migrations run in order.